.github/workflows/master.yml

```diff
@@ -13,76 +29,55 @@ on:
 jobs:
 
   build:
-    name: Build
+    name: Test
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
+        java:
+          - 8
+          - 11
+        spark:
+          - '3.0'
+          - '3.1'
+          - '3.2'
```

Comment:
minor style point: since it's too short, `['3.0', '3.1', '3.2']` looks better

Reply:
the current style looks better to me, many other `short` places use this style too, and it's easy to maintainance for changing